src/main/java/com/eleks/academy/whoami/core/state/ProcessingQuestion.java

```diff
@@ -39,13 +43,39 @@ public void ask(String player, PlayerAction.Action question) {
 		throw new GameException("Not implemented");
 	}
 
-	;
+	public void answer(String player, String value) {
+		if (turn.getCurrentTurn().get(0).getValue() != null) {
+			if (isAnswerState()) {
```

Comment:
you can add into turn method such as `boolean isAnswerer(String player)`

Reply:
In this method use your suggestion.
Firstly I think that I need to use loop but I go to LIst documentation and mention that I can use indexOf instead of loop. I think it is better.